tests/cpp/core.cc

```diff
@@ -28,8 +28,16 @@ struct driver_state {
   buf_type xs;
   static inline const char* name = "driver";
   void reset() {
-    xs = data_msgs({{"a", 0}, {"b", true}, {"a", 1}, {"a", 2}, {"b", false},
-                    {"b", true}, {"a", 3}, {"b", false}, {"a", 4}, {"a", 5}});
+    xs = data_msgs({{"a", 0},
```

Comment:
Weird. i wonder why in every other case of a construct like this, it compacted the list into fewer lines, but in this case it expanded it.

Reply:
I wondered about this particular formatting change as well, because `clang-format` usually seems to try and minimize line count. Though it's well readable that way, IMHO.